src/Extension/CoreExtension.php

```diff
@@ -482,12 +480,12 @@ public static function sprintf($format, ...$values)
      *
      * @internal
      */
-    public static function dateConverter(Environment $env, $date = null, $timezone = null)
+    public function dateConverter($date = null, $timezone = null)
```

Comment:
If the goal is to make this method public, I would suggest renaming it to `convertDate()` to have a proper verb-based naming for it.